draft-ietf-dnsop-structured-dns-error.md

```diff
@@ -417,6 +416,53 @@ via {{?I-D.reddy-add-resolver-info}} and the client includes the EDE OPT
 pseudo-RR in the query. This allows the server to differentiate
 EDE-aware clients from EDE-unaware clients and respond appropriately.
 
+# New Sub-Error Codes Definition
+
+Subsequent documents defining new Sub-Error codes MUST be accepted by
+IETF, MUST also provide a clear definition of the error code (in their
+Reference text), and MUST also provide more characterization than the
```

Comment:
```suggestion
Meaning or Reference clauses), and MUST also provide more characterization than the
```

Reply:
I have removed my text which was intended as guidance to the expert reviewer.